src/js/owl.hash.js

```diff
@@ -79,9 +79,14 @@
 
 		// register event listener for hash navigation
 		$(window).on('hashchange.owl.navigation', $.proxy(function(e) {
-			var hash = window.location.hash.substring(1),
+			var hashes = window.location.hash.substring(1).split(','),
 				items = this._core.$stage.children(),
+				hash,
+				position;
+			for(var i in hashes){
+				hash = hashes[i];
 				position = this._hashes[hash] && items.index(this._hashes[hash]);
+			}
```

Comment:
How does this actually support multiple parameters in the hash? This code looks like it would only ever support the last parameter.